### PR TITLE
Remove bidirectional patterns

### PR DESCRIPTION
**Pull request by SatjaSivcev**
Support for bidirectional patterns is dropped in Zivid 2.0.0

### CODE REVIEW

File: source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<hdevelop file_version="1.2" halcon_version="19.11.0.0">
+<hdevelop file_version="1.2" halcon_version="19.05.0.0">
```
Comment (eskaur) [inline source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev:2]: Why this downgrade?

Comment (SatjaSivcev) [reply source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev]: Removed it. Halcon auto saves the version I use and I have the older version installed.

**Comment by SatjaSivcev** [comment]
Please review again @eskaur

File: source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev [hunk 2]
```diff
@@ -50,7 +50,6 @@
 <l>set_framegrabber_param (AcqHandle, 'Brightness', 1)</l>
 <l>set_framegrabber_param (AcqHandle, 'Gain', 1)</l>
 <c></c>
-<l>set_framegrabber_param (AcqHandle, 'PatternsSineBidirectional', 0)</l>
```
Comment (eskaur) [inline source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev:53]: There are similar lines further down in this file.

Comment (eskaur) [reply source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev:53]: Just search the whole repo for "bidirectional" :)

Comment (SatjaSivcev) [reply source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev:53]: Done, I was testing with the wrong version. Should be ok now.

File: source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev (hunk 1 above)
Comment (eskaur) [reply source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev:2]: Doesn't look like you removed it.

File: source/hdev/Camera/Basic/CaptureHDRLoop.hdev [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<hdevelop file_version="1.2" halcon_version="19.11.0.0">
+<hdevelop file_version="1.2" halcon_version="19.05.0.0">
```
Comment (eskaur) [inline source/hdev/Camera/Basic/CaptureHDRLoop.hdev:2]: Same here

File: source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev (hunk 2 above)
Comment (eskaur) [reply source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev:53]: Looks good. No more bidirectional references :)

**Pull request merged**